apps/api/src/server/applications/examination-timetable-type/examination-timetable-type.controller.js

```diff
@@ -0,0 +1,31 @@
+import * as examinationTimetableTypesRepository from '../../repositories/examination-timetable-types.repository.js';
+import { getCache, setCache } from '../../utils/cache-data.js';
+import { mapExaminationTimetableType } from '../../utils/mapping/map-examination-timetable-type.js';
+
+/**
+ *
+ * @param {import('@pins/api').Schema.ExaminationTimetableType[]} examinationTimetableTypes
+ * @returns {{name: string, displayNameEn: string, displayNameCy: string}[]}
+ */
+const mapExaminationTimetableTypes = (examinationTimetableTypes) => {
+	return examinationTimetableTypes.map((examinationTimetableType) =>
+		mapExaminationTimetableType(examinationTimetableType)
```

Comment:
mapExaminationTimetableType returns => ['id', 'name', 'displayNameEn', 'displayNameCy'] which is the original object structure

Reply:
good call, map fns removed